docs/solutions/k8s-on-dcos.md

```diff
@@ -0,0 +1,271 @@
+
+### 2.2 Case study 2: Gaming
+
+In this case study, we look at a Web 2.0 property which has its main line of business in online games deployed through Web sites and mobile applications. The two main challenges the gaming company faced was to ensure a high utilization of their on-premise cluster and being flexible concerning deployment options. Essentially, they wanted to have game engines built in Kubernetes and also run analytics (Spark, Hadoop, Storm) together on the same cluster in order to lower their TCO (Total Cost of Ownership) and increase their RoI (Return on Investment). As they rolled out the system, Kubernetes on DCOS helped them to create a reliable and flexible architecture, allowing them to triple their compute utilization.
+
+## 3. Solution
+
+Let us now have a look at how Kubernetes is integrated with the Mesosphere DCOS from a 50,000 ft. view:
+
+![Kubernetes on DCOS architecture overview](k8s-on-dcos/k8s-on-dcos_arch_overview.png)
+
+Kubernetes on DCOS begins with a Mesosphere DCOS cluster built atop bare metal or virtual machines running any modern version of Linux. The nodes can either be on-premise or in a public cloud/IaaS setting, such as Google Compute Platform. The Mesosphere DCOS is built around Apache Mesos forms the distributed systems kernel that abstracts away all of the underlying nodes so that they combine to form what looks like one big machine.
+
+The orchestration of apps and services can be done using Kubernetes installed on the DCOS. These Kubernetes workloads can run on the same cluster, and even co-reside on the same machines, as other DCOS workloads, such as those launched with the Marathon or Chronos services.
+
+The DCOS UI and CLI offer a broad range of functionality for end-to-end application lifecycle management and operations including monitoring, upgrading, decommissioning and SLA-related service tasks.
+
+From a technical point of view the [Kubernetes on DCOS](https://github.com/mesosphere/kubernetes-mesos/blob/master/docs/README.md) integration looks as follows:
+
+![Kubernetes-Mesos integration](k8s-on-dcos/k8s-mesos_integration.png)
+
+Comparing the standard Kubernetes deployment with the Kubernetes on DCOS deployment gives the following:
+
+| Aspect                  | Standard Kubernetes | Kubernetes on DCOS |
+| ----------------------- | ------------------- | ------------------ |
+| **multi-service**       | Kubernetes only     | Yes. Kubernetes co-exists with multiple services on the same cluster, including Spark, Cassandra, Marathon, Chronos, Hadoop, Kafka, Jenkins and many others. |
```

Comment:
Kubernetes also supports multi-service. I think the key distinction here is being able to only run Docker workloads (Kubernetes) vs. being able to run a broader range of workloads. So maybe there is a better word than multi-service. One possibility would be something like

mixed container/non-container workloads | Docker containers only | Yes. Kubernetes co-exists with non-containerized services on the same cluster, including ...